src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/Interchange.java

```diff
@@ -40,19 +40,15 @@
     @Getter(lazy = true)
     private final FreeText freeText = edifactMessage.getFreeText();
     @Getter(lazy = true)
+    private final Optional<PatientIdentifier> patientIdentifier = edifactMessage.getPatientIdentifier();
+    @Getter(lazy=true)
     private final InterchangeTrailer interchangeTrailer = edifactMessage.getInterchangeTrailer();
 
     public List<ToEdifactParsingException> validate() {
 
         return Stream.of((Supplier<? extends Segment>) this::getInterchangeHeader,
             (Supplier<? extends Segment>) this::getMessageHeader,
-            (Supplier<? extends Segment>) this::getReferenceTransactionNumber,
```

Comment:
Was this intentional?

Reply:
Yes, as we agreed Interchange validation will only check segments not related to transactions